Sources/ArcGISToolkit/Common/FlashlightButton.swift

```diff
@@ -76,6 +77,7 @@ struct FlashlightButton: View {
 
 private extension View {
     @ViewBuilder
+    @available(visionOS, unavailable)
```

Comment:
Not a big deal, but we typically sort attributes alphabetically.

```suggestion
    @available(visionOS, unavailable)
    @ViewBuilder
```